rest/resource-server/src/docs/asciidoc/licenses.adoc

```diff
@@ -108,4 +108,22 @@ A `POST` request will import SPDX info.
 include::{snippets}/should_document_import_spdx_info/curl-request.adoc[]
 
 ===== Example response
-include::{snippets}/should_document_import_spdx_info/http-response.adoc[]
\ No newline at end of file
+include::{snippets}/should_document_import_spdx_info/http-response.adoc[]
+
+
+[[resources-admin-download--licenses-archive-file_req]]
+==== Downloading licenses archive folder for admin request
+
+A `GET` request help to download the licenses archive folder for admin user.
+
```

Comment:
Change it to, A `GET` request to download license archive

Reply:
Done.